substrate/client/db/src/lib.rs

```diff
@@ -1562,6 +1572,18 @@ impl<Block: BlockT> Backend<Block> {
 						}
 					}
 				}
+
+				// add the storage updates to the changeset
+				let storage_updates_encoded = serialize(&operation.storage_updates.clone()).unwrap();
+				let mut storage_updates_key = hash.encode();
+				storage_updates_key.extend_from_slice(STORAGE_UPDATE_PREFIX.as_bytes());
+				changeset.inserted.push((storage_updates_key, storage_updates_encoded.clone()));
+
+				let child_storage_updates_encoded = serialize(&operation.child_storage_updates.clone()).unwrap();
+				let mut child_storage_updates_key = hash.encode();
+				child_storage_updates_key.extend_from_slice(CHILD_STORAGE_UPDATE_PREFIX.as_bytes());
+				changeset.inserted.push((child_storage_updates_key, child_storage_updates_encoded));
```

Comment:
`STORAGE_UPDATE_PREFIX` / `CHILD_STORAGE_UPDATE_PREFIX` - are prefixes based on their names.
But they are appended at the end of `storage_updates_key` / `child_storage_updates_key`.
In this case prefixes won\t work as a prefix.

Prefixes must be prepended to the keys.
I checked this file - seems like in all other cases prefixes are prepended to the keys.